app/components/table-message/some-result.hbs

```diff
@@ -0,0 +1,10 @@
+{{!-- Partial Result Message --}}
+<TableMessage>
+    <p class="some-result">
+        Als u de persoon die u zoekt niet vindt, kijk na op spelfouten of probeer een andere zoekopdracht. 
+        Indien de persoon ontbreekt, voeg een nieuwe persoon toe
+        <a href="mailto:{{config "contactEmail"}}" class="au-c-link">
+            + {{config "contactEmail"}}
```

Comment:
`voeg een nieuwe persoon toe` should be in the link instead of `+ {{config "contactEmail"}}`